jobs/ml-ops/container/terraform/providers.tf

```diff
@@ -0,0 +1,6 @@
+provider "scaleway" {
+  region = var.region
+  access_key = var.access_key
+  secret_key = var.secret_key
+  project_id = var.project_id
+}
```

Comment:
If you don't specify anything, it will use your config file / environment variables, which is often how you want it setup

Reply:
We need to specify the access/secret key to give to the jobs/container to access the private S3 bucket, so it's probably less confusing to use the same key for provisioning.